src/modules/curations/selectors.ts

```diff
@@ -0,0 +1,29 @@
+import { isThirdPartyCollection } from 'modules/collection/selectors'
+import { Collection } from 'modules/collection/types'
+import { RootState } from 'modules/common/types'
+import { createSelector } from 'reselect'
+import { getCurationsByCollectionId } from './collectionCuration/selectors'
+import { CollectionCuration } from './collectionCuration/types'
+import { getItemCurationsByCollectionId } from './itemCuration/selectors'
+import { ItemCuration } from './itemCuration/types'
+import { CurationStatus } from './types'
+
+export const getHasPendingCuration = createSelector<
```

Comment:
WIP: Not sure if this "generic" method will be used or not. Since the detail page are different, we might not need a generic method since we already know if the collection is TP or not.

Reply:
I ended up deleting this file, it wasn't necessary. It's a good idea tho, if at some point we need something similar to that selector, we can come back to this.